### PR TITLE
Feature/fyp 7/sidebar component

### PR DESCRIPTION
**Pull request by juanCruzSalina**
# Description

Created a new component Sidebar, added the Links components, and moved it into the molecules folder.  Added the stpryboook where I performed all respective tests.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by FrancoMartinetto — APPROVED** [review]
Veo varios problemas con el texto,sobre todo cuando se agranda,pero que Maxi lo revise bien al codigo

File: src/components/molecule/Sidebar/Sidebar.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  title: 'Components/Molecules/Sidebar',
+  component: Sidebar,
+} as ComponentMeta<typeof Sidebar>;
+
+const Template: ComponentStory<typeof Sidebar> = (args) => (
+  <Sidebar {...args} />
+);
+export const Default = Template.bind({});
+Default.args = {
+  data: [
```
Comment (MaxHMoyano) [inline src/components/molecule/Sidebar/Sidebar.stories.tsx:16]: El nombre para la `data` deberia ser `items`. Asi se entiende mejor a que se refiere.

File: src/components/molecule/Sidebar/Sidebar.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+      icon: faUserFriends,
+      text: 'example',
+      color: '#F8F7F9',
+      onHoverColor: '#FE8F29',
```
Comment (MaxHMoyano) [inline src/components/molecule/Sidebar/Sidebar.stories.tsx:21]: Asegurate de usar los colores del tema global y no cualquier hex color que podria no ser parte del tema

File: src/components/molecule/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  list-style: none;
+`;
+
+const NavItem = styled.li`
+  width: 90%;
+  height: 10%;
+  display: flex;
+  justify-content: left;
+  align-items: center;
+`;
```
Comment (MaxHMoyano) [inline src/components/molecule/Sidebar/Sidebar.tsx:36]: Tanto el elemento `ul` como el `li` son innecesarios. El wrapper deberia ser un component flex en modo `column` y nos ahorraria el trabajo de renderizar 3 componentes diferentes.

**Review by MaxHMoyano — CHANGES_REQUESTED** [review]

**Review by MaxHMoyano — DISMISSED** [review]

**Review by MaxHMoyano — APPROVED** [review]

**Pull request merged**